tests/opnReqTest

```diff
@@ -361,6 +361,30 @@ elif [[ $MACHINE_ID = orion ]]; then
   PTMP=$dprefix/stmp
   SCHEDULER=slurm
 
+elif [[ $MACHINE_ID = hercules ]]; then
+
+  module load contrib rocoto/1.3.5
+  ROCOTORUN=$(which rocotorun)
+  ROCOTOSTAT=$(which rocotostat)
+  ROCOTOCOMPLETE=$(which rocotocomplete)
+
+  module use /work/noaa/epic/role-epic/spack-stack/modulefiles
+  module load ecflow/5.8.4-hercules
+  ECFLOW_START=/work/noaa/epic/role-epic/spack-stack/ecflow-5.8.4-hercules/bin/ecflow_start.sh
+  ECF_PORT=$(( $(id -u) + 1500 ))
+
+  QUEUE=windfall
+  COMPILE_QUEUE=windfall
+  PARTITION=hercules
+  dprefix=/work2/noaa/epic/${USER}
+  DISKNM=/work/noaa/nems/emc.nemspara/RT
```

Comment:
@ulmononian can you update with /work/noaa/epic/hercules/UFS-WM_RT

Reply:
done.